.checkstyle/java.header

```diff
@@ -0,0 +1,15 @@
+^/\*
+^ \* Copyright (20\d\d-)?20\d\d Red Hat Inc\.
+^ \*
+^ \* Licensed under the Apache License, Version 2\.0 \(the "License"\);
+^ \* you may not use this file except in compliance with the License\.
+^ \* You may obtain a copy of the License at
+^ \*
+^ \*     http://www\.apache\.org/licenses/LICENSE-2\.0
+^ \*
+^ \* Unless required by applicable law or agreed to in writing, software
+^ \* distributed under the License is distributed on an "AS IS" BASIS,
+^ \* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied\.
+^ \* See the License for the specific language governing permissions and
+^ \* limitations under the License\.
+^ \*/
```

Comment:
The header we use for the `strimzi-kafka-operator` project is simply:

```
/*
 * Copyright 2019, Strimzi authors.
 * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
 */
```

Is there any reason why you're using a different one here?

Reply:
There is already created an issue to change the header for the simpler one. I think changing it should be done in separate PR. https://github.com/strimzi/strimzi-kafka-bridge/issues/151